libs/designsystem/src/lib/components/segmented-control/segmented-control.component.ts

```diff
@@ -92,6 +92,15 @@ export class SegmentedControlComponent {
     this.isSmallSize = size === 'sm';
   }
 
+  private _disableChangeOnSwipe: boolean = false;
+  get disableChangeOnSwipe(): boolean {
+    return this._disableChangeOnSwipe
```

Comment:
There is a pretty well hidden error message in the CI build about a missing semi-colon here. For some reason this was not automatically fixed by prettier on your end (maybe we are not forcing the prettier cli to run at the correct time?).